mainnet/pirin-1/protocols/osmosis-osmosis-osmo.json

```diff
@@ -0,0 +1,45 @@
+{
+    "dex": "osmosis",
+    "dex_network": "OSMOSIS",
+    "lpn_ticker": "OSMO",
+    "stable_currency_ticker": "USDC_NOBLE",
+    "lease_currencies_tickers": [
+        "USDC_NOBLE"
+    ],
+    "payment_only_currencies_tickers": [
+        "ALL_SOL",
+        "ST_ATOM",
+        "ATOM",
+        "ST_OSMO",
+        "WETH",
+        "AKT",
+        "INJ",
+        "AXL",
+        "ALL_BTC",
+        "ALL_ETH"
+    ],
+    "swap_pairs": {
+        "ATOM": [
+            "OSMO",
+            "ST_ATOM"
+        ],
+        "ALL_BTC": [
+            "USDC_NOBLE"
```

Comment:
Redundant because `ALL_BTC`, nor `ALL_SOL`, are connected to `OSMO` or it's descendants.